test/gutenberg-test-themes/behaviors-enabled/theme.json

```diff
@@ -0,0 +1,6 @@
+{
+	"version": 2,
+	"behaviors": {
+		"lightbox": true
```

Comment:
I think it's fine that the labels match the ones we use in blocks. It follows an existing pattern (see block supports) and will probably be easier for folks to grok.